### PR TITLE
PMM-7487 HAProxy connection check.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1514-a2ed4a1
client docker - perconalab/pmm-client-fb:PR-1514-a2ed4a1
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-a2ed4a1.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1514-a2ed4a1&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-a2ed4a1.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/5991/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/5408/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/6961/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1514-a384b26
client docker - perconalab/pmm-client-fb:PR-1514-a384b26
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-a384b26.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1514-a384b26&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-a384b26.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1514-c77f992
client docker - perconalab/pmm-client-fb:PR-1514-c77f992
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-c77f992.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1514-c77f992&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-c77f992.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/5994/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/5411/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/6964/

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/5995/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/5412/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/6965/

File: .gitmodules [hunk 1]
```diff
@@ -63,6 +63,10 @@
 	path = sources/qan-api2/src/github.com/percona/qan-api2
 	url = https://github.com/percona/qan-api2
 	branch = master
+[submodule "qan-app"]
```
Comment (BupycHuk) [inline .gitmodules:66]: @JiriCtvrtka your build is failing because of this.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1514-a57807f
client docker - perconalab/pmm-client-fb:PR-1514-a57807f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-a57807f.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1514-a57807f&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-a57807f.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/6002/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/5419/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/6975/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1514-e70fd14
client docker - perconalab/pmm-client-fb:PR-1514-e70fd14
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-e70fd14.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1514-e70fd14&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1514-e70fd14.tar.gz

**Pull request closed without merging**